framework/base/Module.php

```diff
@@ -705,4 +705,58 @@ public function afterAction($action, $result)
         $this->trigger(self::EVENT_AFTER_ACTION, $event);
         return $event->result;
     }
+
+    /**
+     * Layout find in parent module.
+     * @return array [module, layout]
+     */
+    private function findLayout()
+    {
+        $module = $this;
+
+        do {
+            if (is_string($module->layout)) {
+                return [$module, $module->layout];
+            }
+        }
+        while (($module = $module->module) !== null);
+
+        return [null, null];
+    }
+
+    /**
+     * Layout find in module.
+     * @param string $layout    Layout name
```

Comment:
this should be called $controllerLayout or $defaultLayout for a better understanding